src/overlay.cpp

```diff
@@ -639,57 +638,35 @@ void init_gpu_stats(uint32_t& vendorID, uint32_t reported_deviceID, overlay_para
             metrics_path = gpu_metrics_path;
             SPDLOG_DEBUG("Using gpu_metrics");
          }
-#ifdef HAVE_LIBDRM_AMDGPU
-         else {
-            int idx = -1;
-            //TODO make neater
-            int res = sscanf(path.c_str(), "/sys/class/drm/card%d", &idx);
-            std::string dri_path = "/dev/dri/card" + std::to_string(idx);
-
-            if (!params.enabled[OVERLAY_PARAM_ENABLED_force_amdgpu_hwmon] && res == 1 && amdgpu_open(dri_path.c_str())) {
```

Comment:
Then no need for `force_amdgpu_hwmon` param

Reply:
Indeed - will drop it.